docs/websocket/tracks.md

```diff
@@ -0,0 +1,90 @@
+# Tracks
+
+Provides the list of tracks from a specified playlist.
+
+## Request
+
+```json
+{
+    "id": 15,
+    "type": "spotcast/tracks",
+    "playlist_id": "37i9dQZF1DXcBWIGoYBM5M" or "spotify:playlist:37i9dQZF1DXcBWIGoYBM5M",
```

Comment:
only show one consistent example. I would prefer we show example with uri. The definition of `playlist_id` is sufficient to provide the rest